spec/controllers/openid_connect/logout_controller_spec.rb

```diff
@@ -98,6 +98,28 @@
           action
         end
       end
+
+      context 'with a bad id_token_hint' do
+        let(:id_token_hint) { { id_token_hint: 'abc123' } }
+        it 'tracks analytics' do
+          stub_analytics
+          errors = {
+            id_token_hint: [t('openid_connect.logout.errors.id_token_hint')],
+            redirect_uri: [t('openid_connect.authorization.errors.redirect_uri_no_match')],
+          }
+
+          expect(@analytics).to receive(:track_event).
+            with(Analytics::LOGOUT_INITIATED,
+                 success: false,
+                 client_id: nil,
+                 errors: hash_including(*errors.keys),
+                 error_details: hash_including(*errors.keys),
+                 sp_initiated: true,
+                 oidc: true)
```

Comment:
looks like we only use the keys of `errors` so we could prob do:
```suggestion
          errors_keys = [:id_token_hint, :redirect_uri]

          expect(@analytics).to receive(:track_event).
            with(Analytics::LOGOUT_INITIATED,
                 success: false,
                 client_id: nil,
                 errors: hash_including(*errors_keys),
                 error_details: hash_including(*errors_keys),
                 sp_initiated: true,
                 oidc: true)
```